### PR TITLE
[DOCS-6426] Added new install path

### PR DESCRIPTION
**Pull request by oliveralfresco2018**
I have added in the new install path plus aligned these pages because they were slightly out of sync.

### CODE REVIEW

File: insight-engine/1.4/install/options.md [hunk 1]
```diff
@@ -2,12 +2,9 @@
 title: Installation options
 ---
 
-There are several options for installing Search and Insight Engine:
+You can install Search and Insight Engine using the distribution zip or Docker Compose. If you use the distribution zip there are three different ways you can install the software, the first is with mutual TLS, the second is without mutual TLS (plain HTTP), and the third is without mutual TLS (HTTP with secret word in request header).
```
Comment (aborroy) [inline insight-engine/1.4/install/options.md:5]: Docker Compose is also supporting that three different ways

File: insight-engine/latest/install/options.md [hunk 2]
```diff
@@ -2,19 +2,17 @@
 title: Installation options
 ---
 
-There are several options for installing Search and Insight Engine:
+You can install Search and Insight Engine using the distribution zip or Docker Compose. If you use the distribution zip there are three different ways you can install the software, the first is with mutual TLS, the second is without mutual TLS (plain HTTP), and the third is without mutual TLS (HTTP with secret word in request header).
```
Comment (aborroy) [inline insight-engine/latest/install/options.md:5]: Docker Compose also supports three different ways...

**Review by aborroy — COMMENTED** [review]
LGTM, minor changes suggested

File: insight-engine/1.4/install/options.md (hunk 1 above)
Comment (oliveralfresco2018) [reply insight-engine/1.4/install/options.md]: adjusted sentence

File: insight-engine/latest/install/options.md (hunk 2 above)
Comment (oliveralfresco2018) [reply insight-engine/latest/install/options.md]: adjusted sentence

**Review by aborroy — APPROVED** [review]
LGTM

**Review by gravitonian — APPROVED** [review]
Built and browsed through the install option page, checked the new install section at the end. Looks Good.

**Pull request merged**